scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -83,6 +84,20 @@ public static BuildTargetResult PublishSdkInstallerFile(BuildTargetContext c)
         }
 
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows, BuildPlatform.OSX)]//, BuildPlatform.Ubuntu)]
+        public static BuildTargetResult PublishSharedFxInstallerFile(BuildTargetContext c)
```

Comment:
Consider refactoring the shared logic that this method has in common with PublishSdkInstallerFile into a common method.